stories/BoardOutlineOffset.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { CadViewer } from "src/CadViewer"
+
+function CustomBoard({ boardOutlineOffset }) {
+  return (
+    <board
+      width="10mm"
+      height="10mm"
+      outline={boardOutlineOffset}
```

Comment:
it doesn't make sense to have a board with width/height and an outline. I also already said to remove at least one of the higher order components